### PR TITLE
2 packages from abenori/satysfi-colorbox at 0.1.0

### PR DESCRIPTION
**Pull request by abenori**
This pull-request concerns:
-`satysfi-colorbox.0.1.0`: A SATySFi package for Colored boxes
-`satysfi-colorbox-doc.0.1.0`: Document of Colorbox

---
* Homepage: https://github.com/abenori/satysfi-colorbox
* Source repo: git+https://github.com/abenori/satysfi-colorbox.git
* Bug tracker: https://github.com/abenori/satysfi-colorbox/issues

---
:camel: Pull-request generated by opam-publish v2.0.3
# Automatic follow-ups
Choose follow-up actions.  Do not write anything after this section.
- ~~Add to snapshot `snapshot-develop`~~ (Inconsistent)
- [x] Add to snapshot `snapshot-develop--1` :: satysfi-colorbox-doc.0.1.0 satysfi-colorbox.0.1.0
- Add to snapshot `snapshot-stable-0-0-4`
- ~~Add to snapshot `snapshot-stable-0-0-5`~~ (Inconsistent)
- ~~Add to snapshot `snapshot-stable-0-0-6`~~ (Inconsistent)
- [x] Add to snapshot `snapshot-stable-0-0-6--1` :: satysfi-colorbox-doc.0.1.0 satysfi-colorbox.0.1.0

### CODE REVIEW

**Comment by na4zagin3** [comment]
I'm sorry for my late reply.  I'm now fixing the CI with fixing Satyrographos.

**Comment by na4zagin3** [comment]
Let me retrigger the CI.

File: packages/satysfi-colorbox-doc/satysfi-colorbox-doc.0.1.0/opam [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+dev-repo: "git+https://github.com/abenori/satysfi-colorbox.git"
+bug-reports: "https://github.com/abenori/satysfi-colorbox/issues"
+depends: [
+  "satysfi" { >= "0.0.6" & < "1.0.0" }
+  "satyrographos" { >= "0.0.2.6" & < "0.0.3" }
+
+  # You may want to include the corresponding library
+  "satysfi-colorbox" {= "%{version}%"}
+  "satysfi-dist"
+
```
Comment (na4zagin3) [inline packages/satysfi-colorbox-doc/satysfi-colorbox-doc.0.1.0/opam:17]: The document files require `enumitem/enumitem`.

```suggestion
  "satysfi-enumitem" {>= "3.0.1" & < "4.0" }

```

**Comment by na4zagin3** [comment]
@abenori can you add dependency on `enumitem` at `colorbox-doc` module in `Satyristes`?

**Comment by abenori** [comment]
I added it. Still a problem?

File: packages/satysfi-colorbox/satysfi-colorbox.0.1.0/opam [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+synopsis: "A SATySFi package for Colored boxes"
+description: """A SATySFi package for Colored boxes"""
+maintainer: "Noriyuki Abe"
+authors: "Noriyuki Abe"
+license: "BSD-2-Clause-Views"
+homepage: "https://github.com/abenori/satysfi-colorbox"
+dev-repo: "git+https://github.com/abenori/satysfi-colorbox.git"
+bug-reports: "https://github.com/abenori/satysfi-colorbox/issues"
+depends: [
+  "satysfi" { >= "0.0.6" & < "1.0.0" }
```
Comment (na4zagin3) [inline packages/satysfi-colorbox/satysfi-colorbox.0.1.0/opam:11]: I'll bump the upper boundary when the new SATySFi is released.

```suggestion
  "satysfi" { >= "0.0.6" & < "0.0.7" }
```

File: packages/satysfi-colorbox-doc/satysfi-colorbox-doc.0.1.0/opam [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+synopsis: "Document of Colorbox"
+description: """Document of Colorbox package"""
+maintainer: "Noriyuki Abe"
+authors: "Noriyuki Abe"
+license: "BSD-2-Clause-Views"
+homepage: "https://github.com/abenori/satysfi-colorbox"
+dev-repo: "git+https://github.com/abenori/satysfi-colorbox.git"
+bug-reports: "https://github.com/abenori/satysfi-colorbox/issues"
+depends: [
+  "satysfi" { >= "0.0.6" & < "1.0.0" }
```
Comment (na4zagin3) [inline packages/satysfi-colorbox-doc/satysfi-colorbox-doc.0.1.0/opam:11]: I'll bump the upper boundary when the new SATySFi is released.

```suggestion
  "satysfi" { >= "0.0.6" & < "0.0.7" }
```

File: packages/satysfi-colorbox-doc/satysfi-colorbox-doc.0.1.0/opam [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+  "satyrographos" { >= "0.0.2.6" & < "0.0.3" }
+  "satysfi-enumitem" {>= "3.0.0" & < "3.1.0" }
```
Comment (na4zagin3) [inline packages/satysfi-colorbox-doc/satysfi-colorbox-doc.0.1.0/opam:13]: As enumitem follows semver, it's recommended to specify only two digits for the upper boundary.

```suggestion
  "satysfi-enumitem" {>= "3.0.0" & < "3.1" }
```

File: packages/satysfi-colorbox/satysfi-colorbox.0.1.0/opam [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+  "satyrographos" { >= "0.0.2.6" & < "0.0.3" }
+
+  # If your library depends on other libraries, please write down here
+  "satysfi-dist"
+  "satysfi-base" { >= "1.4.0" & < "2.0.0" }
```
Comment (na4zagin3) [inline packages/satysfi-colorbox/satysfi-colorbox.0.1.0/opam:16]: As base follows semver, it's recommended to specify only two digits for the upper boundary.

```suggestion
  "satysfi-base" { >= "1.4.0" & < "2.0" }
```

**Comment by na4zagin3** [comment]
Let me retrigger the CI

**Comment by na4zagin3** [comment]
Let me retrigger the CI again

**Comment by na4zagin3** [comment]
Merging.  Thank you!

**Pull request merged**